src/layers/base-layer.js

```diff
@@ -396,7 +396,7 @@ export default class BaseLayer {
     this.willUnmount();
   }
 
-  calculateInstancePickingColors(attribute, numInstances) {
+  calculateInstancePickingColors(attribute, {numInstances}) {
```

Comment:
could you add jsdoc to this function on what is the second param and why we need to unpack numInstances out of it?